pkgs/applications/radio/ax25-tools/default.nix

```diff
@@ -0,0 +1,29 @@
+{ lib
+, stdenv
+, fetchurl
+, libax25
+}:
+
+stdenv.mkDerivation rec {
+  pname = "ax25-tools";
+  version = "0.0.10-rc5";
+
+  buildInputs = [ libax25 ];
+
+  # Due to recent unsolvable administrative domain problems with linux-ax25.org,
```

Comment:
Not sure how much this comment is needed.

Reply:
Id like to keep this for any of the packages that were effected by this move from `linux-ax25.org` to `linux-ax25.in-berlin.de` for the next ~6 months

Reason being is theres still many references to the old URL out in the community and it was hard down for several months during 2022 with no clear communication to those in the amateur radio community to the outage. I hope this note helps others understand this is the new primary source of truth.

https://linux-ax25.in-berlin.de/wiki/Main_Page is where I got the verbiage